examples/12_linearization/analysis_options.yaml

```diff
@@ -1,5 +1,5 @@
 general:
-    folder_output: outputs/IEA_level2
+    folder_output: IEA_level2
```

Comment:
I thought we put everything in outputs/ because it was ignored?

Reply:
That makes sense, though any folder created by the script wouldn't be added to the repo unless it's manually added, is my understanding. I changed this just because if the outputs folder doesn't exist on the system, then the IEA_level2 folder can't be made via `mkdir` because it presumes there's an output folder already.

We could change the `folder_output` here to just `outputs`, keep it as-is, or change the `mkdir` logic within WEIS. Let me know if you have a preference and I can do that.